content/cybersecurity/concepts/nmap/terms/tcp-connect-scan/tcp-connect-scan.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'TCP Connect Scan'
+Description: 'A **TCP Connect Scan** is a type of Nmap scan that establishes a complete connection to the target'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+---
+
+A **TCP Connect Scan** `-sT` is a type of Nmap scan that establishes a complete connection to the target by completing a TCP 3-way handshake. 
+Transmission Control Protocol (TCP) is a connection-oriented protocol used in modern networking, like the HTTP/HTTPS connection used to view this article.
+After Nmap's TCP connect scan completes the three way handshake, Nmap then terminates the connection.
+
+This is not to be confused with another common scan technique, *TCP Syn Scan* `-sS` which does not complete its connection to the target host. Because of this, TCP Connect Scan `-sT` will take longer and require more packets to perform.
```

Comment:
```suggestion
This is not to be confused with another common scan technique, TCP Syn Scan (`-sS`) which does not complete its connection to the target host. Because of this, TCP Connect Scan (`-sT`) will take longer and require more packets to perform.
```